packages/tools/client-debugger/client-debugger-view/src/components/TelemetryView.tsx

```diff
@@ -12,13 +12,18 @@ import {
 } from "@fluentui/react";
 import {
 	tokens,
+	Combobox,
+	ComboboxProps,
 	ToggleButton,
 	DataGridBody,
 	DataGridRow,
 	DataGrid,
 	DataGridHeader,
 	DataGridHeaderCell,
 	DataGridCell,
+	Dropdown as DropDown,
```

Comment:
Is `as DropDown` needed?

Reply:
Yes, I added this because there's a Dropdown component from @fluentui/react and @fluentui/react-components. I wanted to keep the filter components consistent. 
To keep the entire UI consistent, I can update the max events displayed dropdown to also be from @fluentui/react-components

Reply:
Yeah, I'd say let's clean up that now and use the same dropdown everywhere.